### PR TITLE
fix: NtConvertBetweenAuxiliaryCounterAndPerformanceCounter definition

### PR DESCRIPTION
**Pull request by MiroKaku**
ida -> kernelbase.dll

```C
HRESULT __stdcall ConvertAuxiliaryCounterToPerformanceCounter(
        ULONGLONG ullAuxiliaryCounterValue,
        PULONGLONG lpPerformanceCounterValue,
        PULONGLONG lpConversionError)
{
  NTSTATUS Status; // eax
  HRESULT result; // eax
  ULONGLONG v5; // [rsp+30h] [rbp+8h] BYREF

  v5 = ullAuxiliaryCounterValue;
  LOBYTE(ullAuxiliaryCounterValue) = 1;
  Status = NtConvertBetweenAuxiliaryCounterAndPerformanceCounter(
             ullAuxiliaryCounterValue,
             &v5,
             lpPerformanceCounterValue,
             lpConversionError);
  if ( Status == STATUS_NOT_SUPPORTED )
    return E_NOTIMPL;
  if ( Status == STATUS_INVALID_PARAMETER )
    return 0x8000000B;
  result = RtlNtStatusToDosErrorNoTeb(Status);
  if ( result > 0 )
    return (unsigned __int16)result | 0x80070000;
  return result;
}
```

```C
HRESULT __stdcall ConvertPerformanceCounterToAuxiliaryCounter(
        ULONGLONG ullPerformanceCounterValue,
        PULONGLONG lpAuxiliaryCounterValue,
        PULONGLONG lpConversionError)
{
  NTSTATUS Status; // eax
  HRESULT result; // eax
  ULONGLONG v5; // [rsp+30h] [rbp+8h] BYREF

  v5 = ullPerformanceCounterValue;
  Status = NtConvertBetweenAuxiliaryCounterAndPerformanceCounter(0i64, &v5, lpAuxiliaryCounterValue, lpConversionError);
  if ( Status == STATUS_NOT_SUPPORTED )
    return E_NOTIMPL;
  if ( Status == STATUS_INVALID_PARAMETER )
    return 0x8000000B;
  result = RtlNtStatusToDosErrorNoTeb(Status);
  if ( result > 0 )
    return (unsigned __int16)result | 0x80070000;
  return result;
}
```

### CODE REVIEW

File: phnt/include/ntexapi.h [hunk 1]
```diff
@@ -1302,10 +1302,10 @@ NTSYSCALLAPI
 NTSTATUS
 NTAPI
 NtConvertBetweenAuxiliaryCounterAndPerformanceCounter(
-    _In_opt_ PLARGE_INTEGER AuxiliaryCounterValue,
-    _Inout_ PLARGE_INTEGER PerformanceCounterValue,
-    _Out_ PLARGE_INTEGER PerformanceOrAuxiliaryCounterValue,
-    _Out_ PLARGE_INTEGER ConversionError
+    _In_ BOOL ConvertAuxiliaryToPerformanceCounter,
```
Comment (dmex) [inline phnt/include/ntexapi.h:1305]: byte/boolean?

File: phnt/include/ntzwapi.h [hunk 2]
```diff
@@ -830,10 +830,10 @@ NTSYSCALLAPI
 NTSTATUS
 NTAPI
 ZwConvertBetweenAuxiliaryCounterAndPerformanceCounter(
-    _In_opt_ PLARGE_INTEGER AuxiliaryCounterValue,
-    _Inout_ PLARGE_INTEGER PerformanceCounterValue,
-    _Out_ PLARGE_INTEGER PerformanceOrAuxiliaryCounterValue,
-    _Out_ PLARGE_INTEGER ConversionError
+    _In_ BOOL ConvertAuxiliaryToPerformanceCounter,
```
Comment (dmex) [inline phnt/include/ntzwapi.h:833]: byte/boolean?

**Review by dmex — CHANGES_REQUESTED** [review]

File: phnt/include/ntexapi.h (hunk 1 above)
Comment (MiroKaku) [reply phnt/include/ntexapi.h]: Yes, is BOOLEAN

**Review by dmex — APPROVED** [review]

**Comment by dmex** [comment]
🍺🎄❄️

**Comment by MiroKaku** [comment]
> 🍺🎄❄️

🍻🎄🎉

**Pull request merged**